onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -286,56 +290,38 @@ def time_fn(args, fn, inputs):
         outputs = fn(inputs)
         logger.info(outputs)
 
-    input_sync = (  # noqa: E731
-        lambda *kwargs: args.io_binding.synchronize_inputs()
-        if args.device != "cpu" and args.benchmark_type in {"ort-msft", "ort-convert-to-onnx"}  # ORT synchronize
-        else lambda *kwargs: torch.cuda.synchronize()
-        if args.device != "cpu" and torch.cuda.is_available()  # PyTorch synchronize
-        else lambda *kwargs: None  # no-op function
-    )
-
-    output_sync = (  # noqa: E731
-        lambda *kwargs: args.io_binding.synchronize_outputs()
-        if args.device != "cpu" and args.benchmark_type in {"ort-msft", "ort-convert-to-onnx"}  # ORT synchronize
-        else lambda *kwargs: torch.cuda.synchronize()
-        if args.device != "cpu" and torch.cuda.is_available()  # PyTorch synchronize
-        else lambda *kwargs: None  # no-op function
-    )
-
     for _ in warmup_range:
-        input_sync()
         fn(inputs)
-        output_sync()
 
     # Benchmark
-    total_time = 0
+    if args.device != "cpu":
+        torch.cuda.synchronize()
```

Comment:
For ORT module, it is better to call io_binding.synchronize_inputs() than torch.cuda.synchronize().

Reply:
will do, thanks